### ISSUE TITLE
max speed

### ISSUE THREAD

**Issue by drAregSlessew** [Waboodoo/Status-Bar-Tachometer#28]
hi! 
As an enthusiastic user of the app, i noted two things.  Perhaps you can take this into account in a future update.  
I flew at a max speed of about 860 km/h.  That was well indicated!  However, 
(1) the speed in the status bar was max 800 and 
(2) the scale of the meter went up to max 720. 

I know this is not a major issue, but it is something to make the app even better.  
Kind regards, Gerard++

![Screenshot_20240625_135902_Speedometer](https://github.com/Waboodoo/Status-Bar-Tachometer/assets/173921241/f7b7169c-2b74-4f3f-a446-cbf6fcf550a6)

**Comment by Waboodoo** [comment]
ok, wow, I did not expect anyone to ever realistically use the app at such speeds, or that the GPS measurements would even accurately function at this speed. I originally built this app with bikes in mind :P

The number shown in the status bar is capped at 800 due to a technical limitation. Essentially I have to include a separate image for each possible number, and I figured that 800 would be sufficient, but seems I was wrong. I'll see that I can increase this to 999 or something, but there'd always need to be some maximum.

For the gauge, I'm not sure how I can accommodate values that go beyond it's maximum value. I had an idea at some point to make it scale dynamically, so adjust its values if the value goes out of range. Perhaps I'll have to revisit that and play around with that a bit.

I can't make any promises on when or if I'll look into this in the near future, but I'll keep your suggestion in mind.
